lib/Backends/CPU/Transforms.cpp

```diff
@@ -126,6 +190,15 @@ bool CPUBackend::transformPostLowering(Function *F,
       }
     }
 
+    // Replace rowwise quantized FC with cpu-specific version.
+    if (auto *RWQFC = dyn_cast<RowwiseQuantizedFullyConnectedNode>(&node)) {
```

Comment:
Why do we need to replace the generic node with the CPU-specific node?

Reply:
Yes, we don't need to do it. I was thinking adding a new CPU-specific node could clearly show the params differences between the CPU kernel and general rowwise quantized FC. But I will update it.  Thanks!

Reply:
Is this case different from CPUConvDKKC8?  It seems like the same sort of transformation to me.

Reply:
To my point of view, DKKC8 is not always applied. The "regular" conv should be in cpu backend as well. But for this rowwise quantized FC,  only "one" version is in CPU backend. So there is no need to add another node. Could you please let us know if there is any other reason for "DKKC8" cpu specific node @nadavrot ? Thanks!